index.html

```diff
@@ -29,5 +29,116 @@ <h1>I'm Enssah. <br> Glad to see <br> you!</h1>
+                <li class="company">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2013</li>
+            </ul>
+            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="teck">
+                <ul class="teck-list">
+                    <li><button class="mobile-btn" type="submit">HTML</button></li>
+                    <li><button class="mobile-btn" type="submit">CSS</button></li>
+                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                </ul>
+            </div>
+            <button class="see-mobile-btn" type="submit">See Projects</button>
+        </section>
+        <!-- card 3 -->
+        <section class="card3">
+            <div class="mobileimage3"></div>
+            <h4>Tonic</h4>
+            <ul class="mobile">
+                <li class="company">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2013</li>
+            </ul>
+            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="teck">
+                <ul class="teck-list">
+                    <li><button class="mobile-btn" type="submit">HTML</button></li>
+                    <li><button class="mobile-btn" type="submit">CSS</button></li>
+                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                </ul>
+            </div>
+            <button class="see-mobile-btn" type="submit">See Projects</button>
+        </section>
+        <!-- card4 -->
+        <section class="card4">
+            <div class="mobileimage4"></div>
+            <h3>Multi-Post <br> Stories</h3>
+            <ul class="mobile">
+                <li class="company">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2013</li>
+            </ul>
+            <p class="mobile-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="teck">
+                <ul class="teck-list">
+                    <li><button class="mobile-btn" type="submit">HTML</button></li>
+                    <li><button class="mobile-btn" type="submit">CSS</button></li>
+                    <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                </ul>
+            </div>
+            <button class="see-mobile-btn" type="submit">See Projects</button>
+        </section>
+    </section>
+       <!-- about area -->
+        <section class="about">
+            <h5>About Myself</h5>
+            <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <span>LET'S CONNECT</span>
```

Comment:
- [ ] For semantics and SEO reasons, kindly use any of the header tag aside `h1`. Thanks. 🙏